### PR TITLE
CRM-40 feature

### PR DESCRIPTION
**Pull request by nsSlalom**

### CODE REVIEW

File: src/classes/ContactListCreationController.cls [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  Modification Log    :
+ * Developer                    Date                    Description
+ * ----------------------------------------------------------------------------
+ *   Nick Serafin/Slalom       05/09/2016                Original Version
+ *************************************************************************************/
+
+public with sharing class ContactListCreationController {
+
+	public Id accountId{get;set;} 
+	public ContactTeamWrapper AccountTeamMemberRecord;
```
Comment (rbonslater) [inline src/classes/ContactListCreationController.cls]: @nsSlalom can we change all of the variables that begin with upper case to lower case?

File: src/classes/ContactListCreationController.cls [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+			}
+
+			for (ContactTeamNonSFWrapper teamNonSFWrapperObject : listOfAccountTeamNonSFMembers) {
+
+				if (teamNonSFWrapperObject.isSelected == true) {
+					selectedAccountTeamNonSFMembers.add(teamNonSFWrapperObject.accountTeamNonSFObj);
+				}
+			}
+
+			if (!selectedAccountTeamMembers.isEmpty() && !selectedAccountTeamNonSFMembers.isEmpty()) {
```
Comment (rbonslater) [inline src/classes/ContactListCreationController.cls]: @nsSlalom couldn't we just do this:

if (!selectedAccountTeamMembers.isEmpty()) {
	for(AccountTeamMember accountTeamObj : selectedAccountTeamMembers){
		accountTeamMemberIds.add(accountTeamObj.User.Id);
        }
}

if (!selectedAccountTeamNonSFMembers.isEmpty()) {
	for(Account_Team_Non_SF__c accountTeamNonSFObj : selectedAccountTeamNonSFMembers){
		accountTeamNonSFMemberIds.add(accountTeamNonSFObj.Team_Member_Name__c);
	}				
} 
return null;

Or is there a reason why we have so many combinations of IF ELSE statements?  Or I may be missing something.

File: src/classes/ContactListCreationControllerTest.cls [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+* Developer                        Date                  
+* ----------------------------------------------------------------------------                 
+* Nick Serafin/Slalom             5/11/2016
+*************************************************************************************/
+@isTest
+private class ContactListCreationControllerTest {
+
+	@isTest static void selectedContactsTest() {
+
+		Account TestAcnt = TestClassUtiltity.CreateAccountRecord('TestAccount');
```
Comment (rbonslater) [inline src/classes/ContactListCreationControllerTest.cls]: @nsSlalom I'm sure you saw the misspelling of this class TestClass**Utiltity**.  We'll put this on our tech debt list.

Comment (nsSlalom) [inline src/classes/ContactListCreationControllerTest.cls]: @rbonslater yeah its a known issue

File: src/pages/ContactListCreation.page [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+            accountTeam.push('\'{!team}\'');
+          </apex:repeat>
+          <apex:repeat value="{!accountTeamNonSFMemberIds}" var="teamNonSF">
+            accountTeamNonSF.push('\'{!teamNonSF}\'');
+          </apex:repeat> 
+
+          var accountTeamIdsQuery = accountTeam.join("|");     
+          var accountTeamNonSFIdsQuery = accountTeamNonSF.join("|");
+
+          var CongaURL = "https://composer.congamerge.com" +
```
Comment (rbonslater) [inline src/pages/ContactListCreation.page]: @nsSlalom we could have created a custom metadata type for the Conga settings.  Do you have time to do this?
CongaURL = https://composer.congamerge.com
DefaultPDF = 1
DS7 = 3

Comment (nsSlalom) [inline src/pages/ContactListCreation.page]: yeah I'll do that right now

File: src/pages/ContactListCreation.page [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+
+        function openConga() { 
+
+            var accountTeam = new Array();
+            var accountTeamNonSF = new Array();
+            var templateId = '{!templateId}';
+            var accountTeamQueryId = '{!acccountTeamQueryId}';
+            var accountTeamNonSFQueryId = '{!acccountTeamNonSFQueryId}';
+
+          <apex:repeat value="{!accountTeamMemberIds}" var="team">
```
Comment (rbonslater) [inline src/pages/ContactListCreation.page]: @nsSlalom this is cool, using an apex repeater in JavaScript context to build arrays.

**Comment by rbonslater** [comment]
@nsSlalom did my comments make sense?  Did you already make the changes or are you still working on it?

**Comment by nsSlalom** [comment]
@rbonslater yeah I added the custom metadata type to hold the conga ids I needed in my last commit

**Comment by nsSlalom** [comment]
@rbonslater I made those changes

**Comment by rbonslater** [comment]
@nsSlalom thanks Nick.  This code looks great.
@fatimasana

**Comment by fatimasana** [comment]
Looks good to me. @SlalomAK will be doing the deploy to SAMDEVGRP.

**Pull request merged**